obs-browser-plugin.cpp

```diff
@@ -388,6 +388,12 @@ static void BrowserInit(void)
 
 	app = new BrowserApp(tex_sharing_avail);
 
+	char *pagesPath = obs_module_file("dev_tools_discovery.html");
+	char *pagesFile = os_quick_read_utf8_file(pagesPath);
```

Comment:
Could these be BPtr's instead (making the explicit bfree calls unneeded)?

Reply:
Ah, I always forget about BPtr. Yes, that works here.